integration_tests/tests/test_all_columns_anomalies.py

```diff
@@ -95,13 +95,13 @@ def test_all_columns_anomalies_with_where_expression(
 
     params = dict(DBT_TEST_ARGS, where="universe = 'Marvel'")
     test_results = dbt_project.test(
-        test_id, DBT_TEST_NAME, params, multiple_results=True
+        f"{test_id}_marvel", DBT_TEST_NAME, params, multiple_results=True
```

Comment:
Needed to change the ID, because after the fixes to the backfill it only updates the last 2 dates